App.tsx

```diff
@@ -19,6 +20,7 @@ const Stack = createStackNavigator<RootStackParamList>();
 enum Routes {
   HOME = 'Home',
   DETAILS = 'Details',
+  STATIONSELECT = 'StationSelect',
```

Comment:
These enum values are typically "screaming snake case", which `LOOKS_LIKE_THIS`.

So this line should be `STATION_SELECT = 'StationSelect',`